lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -411,9 +517,33 @@ def start(self):
             )
             self.log.info("PSF HDU created")
 
+        if self.energy_dependent_gh:
+            gh_header = self.hdus[1].header.copy()
+            gh_header["HDUCLAS2"] = "GH_CUTS"
```

Comment:
This is not defined in GADF, please don't add headers that make it look like it is a specified GADF HDU

Reply:
As I need to fill the DL3 Index table, can I fill the headers for this HDU in the following way,
`"CREATOR" = pyirf {pyirf.__version__}
"HDUDOC" = "N/A"
"HDUVERS" = "N/A"
"HDUCLASS" = "N/A"
"HDUCLAS1" = "RESPONSE"
"HDUCLAS2" = "GH_CUTS"
"HDUCLAS4" = "GH_CUTS_2D"
`
Or some other values?

Reply:
And, maybe we should add this HDU information in GADF? If it seems ok with you, I can at least open an issue there

Reply:
Why do you need to fill this into the hdu index table? It is not a GADF HDU and thus not used by gammapy

Reply:
And creator should be `lstchain {lstchain.__version__}`